BookingsApi/BookingsApi/Controllers/HearingsController.cs

```diff
@@ -115,7 +115,30 @@ public async Task<IActionResult> GetHearingsByUsername([FromQuery]string usernam
             return Ok(response);
         }
 
-        
+        /// <summary>
+        /// Get list of all hearings in a group
+        /// </summary>
+        /// <param name="groupId">the group id of the single day or multi day hearing</param>
+        /// <returns>Hearing details</returns>
+        [HttpGet(Name = "GetHearingsByGroupId")]
+        [OpenApiOperation("GetHearingsByGroupId")]
+        [ProducesResponseType(typeof(List<HearingDetailsResponse>), (int) HttpStatusCode.OK)]
+        public async Task<IActionResult> GetHearingsByGroupId([FromQuery] Guid groupId)
+        {
+            var query = new GetHearingsByGroupIdQuery(groupId);
+            var hearings = await _queryHandler.Handle<GetHearingsByGroupIdQuery, List<VideoHearing>>(query);
+
+            if (hearings.Count < 1)
```

Comment:
The query is always returning a list, even if it's empty, the remainder of the code is fine.